README.md

```diff
@@ -1,30 +1,39 @@
-# Back End Test Project
+# Back End Test Project <img src="https://coderockr.com/assets/images/coderockr.svg" align="right" height="50px" />
+
 You should see this challenge as an opportunity to create an application following modern development best practices (given the stack of your choice), but also feel free to use your own architecture preferences (coding standards, code organization, third-party libraries, etc). It’s perfectly fine to use vanilla code or any framework or libraries.
 
 ## Scope
-In this challenge you should build an API for an application such as a social event network that implements the following features:
+
+In this challenge you should build an API for an application that stores and manages investments, it should have the following features:
+
+  1. __Creation__ of an investment with a owner, a creation date and an amount.
+    1. An investment can be in the past.
+    2. An investment should not be or became negative.
+  2. __View__ of a investment with its withdraws and current balance.
+    1. Current balance should be the sum of the invested amount, the [interest] and withdraws.
+  3. Partial or total __withdraw__ of the investment amount.
+    1. Should not be able to withdraw more than the balance.
+    2. Withdraws in the past can be created, but can't be before the investment creation or previous withdraws.
+    3. Should show the amount paid on [taxes].
+  4. __List__ of all investments of a person.
```

Comment:
List of a person's investments (com ou sem paginação?)

Reply:
eu ia deixar livre, mas acho válido colocar paginação sim.